python/chemiscope/input.py

```diff
@@ -167,6 +138,45 @@ def write_input(path, frames, meta=None, properties=None, cutoff=None):
 
     This function uses :py:func:`create_input` to generate the input data, see
     the documentation of this function for more information.
+
+    Here is a quick example of generating a chemiscope input reading the
+    structures from a file that `ase <ase-io_>`_ can read, and performing PCA
+    using `sklearn`_ on a descriptor computed with another package.
+
+    .. code-block:: python
+
+        from ase import io
+        import numpy as np
+        import sklearn
```

Comment:
just add a `from chemiscope import write_input` in case people directly copy / paste
```suggestion
        import sklearn
        from chemiscope import write_input
```